wordpress-mobile/WordPress-Android
# Issue/5041 disable toolbar buttons

### Fix
Add function to Aztec to allow enabling/disabling format toolbar buttons when post title is focused as described in https://github.com/wordpress-mobile/WordPress-Android/issues/5041.

### Test
0. Wait for https://github.com/wordpress-mobile/WordPress-Aztec-Android/pull/222 to merge.
1. Go to ***Sites*** tab.
2. Tap ***Create*** floating button.
3. Tap ***Title*** area.
4. Notice format toolbar buttons are disabled.
5. Tap ***Tenochtitlan is the hint*** area.
6. Notice format toolbar buttons are enabled.
7. Tap ***HTML*** format button.
8. Notice format toolbar buttons are disabled.
9. Tap ***Title*** area.
10. Notice format toolbar buttons are disabled.
11. Tap ***HTML*** format button.
12. Notice format toolbar buttons are enabled.
